### DISCUSSION TITLE
Anyway to keep using the old UI

### DISCUSSION THREAD

**Discussion by yarkinwho** [blockscout/blockscout#8886] [General]
Tried to update to latest version only to discover the nice new frontend. 

But we had some customization in old UI and need some work to switch to new frontend.

My question is 
1 is there any way to use the old UI in current version? 
2 if not, what's the highest version I can use with the old UI?

Thanks.

**Comment by vbaranov** [comment]
When you run backend, the old UI is still available at 4000 port. It is applicable to all Blockscout versions.
